docker-compose.yml

```diff
@@ -18,6 +18,12 @@ services:
       DATA_PATH: "/covid19/data/"
     restart: always
     command: "uwsgi --uid www-data --gid www-data --socket 0.0.0.0:5000 --die-on-term -w covid_web.wsgi:app"
+  react-web:
+    build: web/react-web/
+    image: react-web
+    volumes:
+      - "react-web:/react-web"
+    command: "bash -c ' rm -rf /react-web/* && cp -a . /react-web/'"
```

Comment:
What does this line do?

Reply:
Dockerfile from react-web just build the web and make static files that resides in image. This line move them to shared volume called `react-web` and docker container `react-web` ends. Shared volume is then accessible by nginx and web can be access at http://localhost:8081/covid19/graphs.

Reply:
Okay, than this container should be called `react-web-builder` instead.

Reply:
Applied